AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -174,6 +178,51 @@ + (void)setCustomProperties:(MSCustomProperties *)customProperties {
 }
 #endif
 
++ (void)setAuthToken:(NSString *)authToken {
+  [[MSAppCenter sharedInstance] setAuthToken:authToken];
+}
+
+- (void)setAuthToken:(NSString *)authToken {
+  MSAuthTokenContext *authTokenContext = [MSAuthTokenContext sharedInstance];
+  MSJwtClaims *claims = nil;
+  if (authToken != nil) {
+    claims = [MSJwtClaims parse:authToken];
```

Comment:
Error message in `MSJwtClaims` is a dead code. Please remove `nil` check here or move the log message from `MSJwtClaims` to here.